src/v/cloud_topics/app.cc

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "cloud_topics/app.h"
+
+#include "cloud_topics/reconciler/reconciler.h"
+
+namespace experimental::cloud_topics::reconciler {
```

Comment:
Just wondering if this should be some `cloud_topics::app`, or is this really only going to encompass the reconciler?

Reply:
too early to tell i think. most likely the "app" will split into finer grained things. same question arising when i'm doing this for kafka/server layer.